compiler/test/formatter_outputs/ignores.gr

```diff
@@ -11,10 +10,10 @@ let y = [3, 4, 5]
 if (true) {
   // formatter-ignore
   let yy = [1,
-  2]  // comment
+  2] // comment
 
   let zz = [5, 6]
 }
 
 // formatter-ignore
-      let single_ast_line = "x"
+let single_ast_line = "x"
```

Comment:
Why did this change if it uses `// formatter-ignore`?

Reply:
Keen eye!    There was a good reason for it to do with locations, I'll have another look though

Reply:
Oh, because it's really hard (impossible) to add the spacing back.   We don't know how much padding the formatter is adding around where we are inserting.  So you can see it works in the line above but not here.  I can't see how to fix it presently.

Reply:
I figured this was the best of a poor choice